builtin/bisect--helper.c

```diff
@@ -743,6 +743,31 @@ static int bisect_start(struct bisect_terms *terms, int no_checkout,
 	string_list_clear(&states, 0);
 	strbuf_release(&start_head);
 	strbuf_release(&bisect_names);
+	if (res)
+		return res;
+
+	res = bisect_auto_next(terms, NULL);
+	/*
+	 * In case of mistaken revs or checkout error, or signals received,
+	 * "bisect_auto_next" below may exit or misbehave.
+	 * We have to trap this to be able to clean up using
+	 * "bisect_clean_state".
```

Comment:
I think the comment could be updated to remove the part about signals as we are not doing anything anymore to trap signals. While at it, maybe "trap" should be changed to "catch" or perhaps better "handle". As "trap" is a reference to the shell "trap" builtin, which isn't used anymore.